org.audiveris.audiveris.yml

```diff
@@ -0,0 +1,982 @@
+
+app-id: org.audiveris.audiveris
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+build-options:
+  append-path: "/usr/lib/sdk/openjdk17/bin"
+command: Audiveris
+finish-args:
+  # Network access
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  # Calling abort() on malloc-check doesn't fly with OpenJDK
+  - --env=MALLOC_CHECK_=0
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: audiveris
+    buildsystem: simple
+    build-commands:
+      - ln -f gradle-7.3-all.zip gradle/wrapper/gradle-7.3-all.zip
+      - sed -Ei 's,https\\://services.gradle.org/distributions/,,' gradle/wrapper/gradle-wrapper.properties
+      - sh ./mkgradlerepo.sh
+      - install -m 0644 -D -t /app/share/tessdata *.traineddata
+      - install -m 0644 -D res/icon-256.png /app/share/icons/hicolor/256x256/apps/org.audiveris.audiveris.png
+      - install -m 0644 -D res/icon-64.png /app/share/icons/hicolor/64x64/apps/org.audiveris.audiveris.png
+      - install -m 0644 -D res/icon-50.png /app/share/icons/hicolor/50x50/apps/org.audiveris.audiveris.png
+      - install -m 0644 -D -m 644 org.audiveris.audiveris.desktop /app/share/applications/org.audiveris.audiveris.desktop
+      - install -m 0644 -D -m 644 org.audiveris.audiveris.metainfo.xml /app/share/metainfo/org.audiveris.audiveris.metainfo.xml
+      - ./gradlew --offline build
+      - tar x -C /app --strip-components=1 -f build/distributions/Audiveris-*.tar
+      - sed -i -f add-tessdata-prefix.sed /app/bin/Audiveris
+    sources:
+      - type: git
+        url: https://github.com/Audiveris/audiveris
+        branch: 5.3.1
```

Comment:
never a branch. this should be a commit and/or a tag.

Reply:
ok.